Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM mhart/alpine-node:6.14.2
 
-RUN apk update && apk upgrade && apk add --no-cache git
+RUN apk add --no-cache git
```

Comment:
What's the reason for this change?

Reply:
Hi. It seams that your base image is up to date so i thought it's ok to just install the dependencies without keeping the package cache from "apk update". If you still want to update the whole image, "apk --no-cache upgrade && apk add --no-cache git" would be a nice alternative.

Reply:
Sounds good, can you make the change?

Reply:
Done.